### PR TITLE
ACCUMULO-4750 Created caching documentation

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: _docs-2-0/administration/caching.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+order: 11
+---
+
+Accumulo tablet servers have a **block cache** that buffers data in memory to limit reads from disk.
+This caching has the following benefits:
+
+* reduces latency when reading data
+* helps alleviate hotspots in tables
+
+The block cache stores index and data blocks. A typical Accumulo read will perfrom a binary search
```
Comment (ctubbsii) [inline _docs-2-0/administration/caching.md]: What is "a typical Accumulo"?

Comment (ctubbsii) [reply _docs-2-0/administration/caching.md]: Never mind. I got confused with the phrasing. Perhaps "A typical read operation" instead?

File: _docs-2-0/administration/caching.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+over several index blocks followed by a linear scan of one or more data blocks. Each tablet server
+has its own block cache that is shared by all hosted tablets. Therefore, block caches are only enabled
+for tables where read performance is critical.
+
+## Configuration
+
+While the block cache is enabled by default for the Accumulo metadata tables, it must be enabled
+for all other tables by setting the following table properties to `true`:
+
+* [table.cache.block.enable] - enables data block cache on the table
```
Comment (keith-turner) [inline _docs-2-0/administration/caching.md]: This is nice, linking to the prop docs.

File: _docs-2-0/administration/caching.md (hunk 1 above)
Comment (mikewalch) [reply _docs-2-0/administration/caching.md]: Fixed in 4c22f3edc2b6

**Pull request merged**